content/pages/homepage.html

```diff
@@ -112,28 +111,33 @@ <h3 class="ui large monospace header">
 
           <div class="ui centered vertically padded container stackable grid">
 
-            {% call homepage.feature_detail_item(header="Multiple builders", icon="fad fa-terminal") %}
-              Integrated support for Sphinx, MkDocs, and Jupyter Book. Use what works best for your project.
+            {% call homepage.feature_detail_item(header="Supports any doc tool", icon="fad fa-terminal") %}
+              Integrated support for Sphinx, MkDocs, and Jupyter Book.
+              Flexible enough to work with any documentation tool.
             {% endcall %}
 
-            {% call homepage.feature_detail_item(header="Versioned docs", icon="fad fa-layer-group") %}
-              Match your product release cycle with multiple versions of your docs. Automatically build versions from any branch or tag.
+            {% call homepage.feature_detail_item(header="Integrated search", icon="fad fa-search") %}
+              Easily search all your docs via our dashboard,
+              and offer your users a more powerful search with our  search API.
             {% endcall %}
 
-            {% call homepage.feature_detail_item(header="Build constantly", icon="fad fa-gears") %}
-              Preview every pull request, and build with every commit. Keep your docs up to date, automatically.
+            {% call homepage.feature_detail_item(header="Pull request previews", icon="fad fa-gears") %}
+              Preview each pull request with a unique URL,
+              allowing fast collaboration across your organization.
             {% endcall %}
 
-            {% call homepage.feature_detail_item(header="Integrated search", icon="fad fa-search") %}
-              Help your users find what the need with integrated search across all your projects and versions.
+            {% call homepage.feature_detail_item(header="Versioned docs", icon="fad fa-layer-group") %}
+              Match your product release cycle with multiple versions of your docs. Automatically build versions from any branch or tag.
             {% endcall %}
 
-            {% call homepage.feature_detail_item(header="Private cloning", icon="fad fa-lock", paid=True) %}
-              Clone private repositories using a connected GitHub, GitLab, or Bitbucket account.
+            {% call homepage.feature_detail_item(header="Private repository support", icon="fad fa-lock", paid=True) %}
+              Clone private repositories using a connected GitHub account,
+              or any Git provider with an SSH key.
```

Comment:
Should either mention all or none of the providers here I guess. This sounds like only GitHub is supported.

```suggestion
              Clone private repositories using a connected GitHub, GitLab, or Bitbucket account,
              or clone from any Git provider with an SSH key.
```